README.md

```diff
@@ -99,28 +99,32 @@ It should take less than 15 minutes to install (depends on your machine).
 Customize in `~/.laptop.local`
 ------------------------------
 
-Your `~/.laptop.local` is run at the end of the Laptop script.
+Your `~/.laptop.local` is run near the end of the Laptop script.
```

Comment:
I should have done this in https://github.com/thoughtbot/laptop/pull/432, just made it part of that change.